pkg/controller/component/metad_updater.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2021 Vesoft Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package component
+
+import (
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/vesoft-inc/nebula-operator/apis/apps/v1alpha1"
+	"github.com/vesoft-inc/nebula-operator/pkg/kube"
+	utilerrors "github.com/vesoft-inc/nebula-operator/pkg/util/errors"
+	extenderutil "github.com/vesoft-inc/nebula-operator/pkg/util/extender"
+	"github.com/vesoft-inc/nebula-operator/pkg/util/resource"
+)
+
+type metadUpdater struct {
+	extender  extenderutil.UnstructuredExtender
+	podClient kube.Pod
+}
+
+func NewMetadUpdater(podClient kube.Pod) UpdateManager {
+	return &metadUpdater{
+		extender:  extenderutil.New(),
+		podClient: podClient,
+	}
+}
+
+func (m *metadUpdater) Update(
+	nc *v1alpha1.NebulaCluster,
+	oldUnstruct, newUnstruct *unstructured.Unstructured,
+	gvk schema.GroupVersionKind,
+) error {
+	if *nc.Spec.Metad.Replicas == int32(0) {
+		return nil
```

Comment:
Prohibited to change to `0` ?  And the configurations do not support modification at this time?

Reply:
If the webhook was closed, 0 can be set here.